cni.go

```diff
@@ -183,8 +181,6 @@ func (c *libcni) SetupSerially(ctx context.Context, id string, path string, opts
 	if err := c.ready(); err != nil {
 		return nil, err
 	}
-	c.RLock()
```

Comment:
I don't think this is totally correct (removing the lock), at least it's not clear if it is, see  #126